src/bp/ui-studio/src/web/components/Layout/Sidebar.scss

```diff
@@ -8,6 +8,12 @@
   cursor: pointer;
 }
 
+.logo {
```

Comment:
If you add this, can you in this case remove the padding on the parent div (in the sidebar.jsx file line 168) which was placed there as a quick fix for the exact same reason.